bin/host/src/main.rs

```diff
@@ -0,0 +1,82 @@
+use alloy_primitives::B256;
+use clap::{Parser, ValueEnum};
+use serde::Serialize;
+use std::path::PathBuf;
+use std::str::FromStr;
+
+#[tokio::main]
+async fn main() -> anyhow::Result<()> {
+    let _cli = Cli::parse();
+    tracing_subscriber::Registry::default().init();
```

Comment:
It would be nice to have a tracing subscriber with configurable verbosity for `tracing::Level`s.

Suggested:

1. Add a `v` field to the `Cli` struct, using `ArgAction` to increment (i.e. `-v` up to `-vvvvv`)

```rs
struct Cli {
    /// Verbosity level (0-4)
    #[arg(long, short, help = "Verbosity level (0-4)", action = ArgAction::Count)]
    v: u8,
    // ...
}
```
2. Add a function that can initialize the tracing subscriber with a configured verbosity level:
```rs
/// Initializes the tracing subscriber
///
/// # Arguments
/// * `verbosity_level` - The verbosity level (0-4)
///
/// # Returns
/// * `Result<()>` - Ok if successful, Err otherwise.
fn init_tracing_subscriber(verbosity_level: u8) -> Result<()> {
    let subscriber = tracing_subscriber::fmt()
        .with_max_level(match verbosity_level {
            0 => Level::ERROR,
            1 => Level::WARN,
            2 => Level::INFO,
            3 => Level::DEBUG,
            _ => Level::TRACE,
        })
        .finish();
    tracing::subscriber::set_global_default(subscriber).map_err(|e| anyhow!(e))
}
```
3. Initialize the subscriber in main using the above function, i.e. `init_tracing_subscriber(cli.v)?`